packages/sirius-web/backend/sirius-web-application/src/main/java/org/eclipse/sirius/web/application/library/controllers/LibraryCurrentEditingContextDataFetcher.java

```diff
@@ -0,0 +1,37 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.application.library.controllers;
+
+import graphql.schema.DataFetchingEnvironment;
+import org.eclipse.sirius.components.annotations.spring.graphql.QueryDataFetcher;
+import org.eclipse.sirius.components.graphql.api.IDataFetcherWithFieldCoordinates;
+import org.eclipse.sirius.components.graphql.api.LocalContextConstants;
+
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * Data fetcher for the field Library#currentEditingContext.
+ *
+ * @author mcharfadi
+ */
+@QueryDataFetcher(type = "Library", field = "currentEditingContext")
+public class LibraryCurrentEditingContextDataFetcher implements IDataFetcherWithFieldCoordinates<String> {
+
+    @Override
+    public String get(DataFetchingEnvironment environment) throws Exception {
+        Map<String, Object> localContext = environment.getLocalContext();
```

Comment:
Unless I missed something, there should be no need for a context here to give this information